api/v3/Case/Getstats.php

```diff
@@ -125,14 +129,18 @@ function civicrm_api3_case_getstats(array $params) {
  */
 function _civicrm_api3_case_add_case_category_query_filter($query, array $caseTypes) {
   $caseTypeIds = array_column($caseTypes['values'], 'id');
+  $param = ['IN' => $caseTypeIds];
+
+  if ($caseTypes[0] === 'IS NULL') {
+    $param = ['IS NULL' => []];
+  }
 
-  if (empty($caseTypeIds)) {
+  if (empty($caseTypeIds) && $caseTypes[0] !== 'IS NULL') {
```

Comment:
No need to add the extra condition. If caseTypeId is empty, we should return as usual. We can remove `$caseTypes[0] !== 'IS NULL'`